### PR TITLE
Integrate UI of event details with ticket real data

### PR DESCRIPTION
**Pull request by jean-neiverth**
# Summary

- add new query to get tickets by event id
- use the query in event page details, so now the ticket listing are real
- create hooks to get event data by id and tickets by

# How to test

1. Go to Event > My events
2. Enter a event you own (if you don't have, create one in Event > New event)
- You should be able to see the ticket management section with a button to assign tickets
3. Assign some tickets
- These tickets should appear in the management section

### CODE REVIEW

File: src/app/event/Modal.tsx [hunk 1]
```diff
@@ -45,6 +45,8 @@ export function Modal({ eventId }: { eventId: string }) {
       };
     });
 
+    console.log("Attesting tickets", tickets);
```
Comment (luizmuraro) [inline src/app/event/Modal.tsx]: It is a must on the app or it's just for testing? If it's for testing and you will remove it later, you should add a TODO to remove it later

File: src/app/event/[id]/page.tsx [hunk 2]
```diff
@@ -1,112 +1,70 @@
...
-    () =>
-      data?.attestation && parseEventsData(data.attestation?.decodedDataJson),
-    [data?.attestation]
-  );
+  const { ticketAddresses } = useTicketAddresses({ eventId, eventData });
 
   if (!account) return <p>Please connect wallet to load the informations...</p>;
 
-  if (parsedData === undefined) return <p>loading...</p>;
+  if (eventData === undefined) return <p>loading...</p>;
```
Comment (luizmuraro) [inline src/app/event/[id]/page.tsx]: The loading should not be handled by  `fetchingEvent`? if so you should have something like that:

```suggestion
  if (fetchingEvent) return <p>loading...</p>;
```

File: src/app/event/[id]/page.tsx [hunk 3]
```diff
@@ -1,112 +1,70 @@
...
 
-  if (parsedData === null)
+  if (eventData === null)
     return <p>It seems that this event does not exist.</p>;
 
+  if (ticketAddresses === undefined) return <p>Loading...</p>;
+
```
Comment (luizmuraro) [inline src/app/event/[id]/page.tsx]: It's possible to come null and undefined? The error handling shouldn't be able to handle it?

**Review by luizmuraro — COMMENTED** [review]
Just some questions regarding the null and undefined handling at the event page, but great job at all, creating the hooks and using useQuery 🚀

File: src/app/event/[id]/page.tsx (hunk 2 above)
Comment (jean-neiverth) [reply src/app/event/[id]/page.tsx]: Well observed. I'm trying to solve here but it seems this system is not intelligent enough for this. So if I change this I will start getting ts warnings like "hey, eventData can be undefined"

File: src/app/event/[id]/page.tsx (hunk 3 above)
Comment (jean-neiverth) [reply src/app/event/[id]/page.tsx]: The point is that the backend returns data: {attestation: null} if the id does not exist, and not a error response. My eventData is the data.attestation, so thats why (if undefined -> didnt return) and (if null -> returned bad id)

**Review by luizakp — APPROVED** [review]
Let's merge this!

**Pull request merged**